Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -964,8 +964,29 @@ class AnimatedInterpolation extends AnimatedWithChildren {
     super.__detach();
   }
 
+  __transformDataType(range) {
+    if (!Array.isArray(range)) {
+      return;
+    }
+
+    // Change the string array type to number array
+    // So we can reuse the same logic in iOS and Android platform
+    for (let i = 0; i < range.length; i++) {
+      let value = range[i];
+      if (typeof value !== 'string') {
+        continue;
+      }
+      if (/deg$/.test(value)) {
+        let degree = parseFloat(value.replace(/deg/, ''), 10) || 0;
```

Comment:
There a check for deg already so it should not get executed for a color. For now we don't support colors anyway.